sdk/core/Azure.Core/src/Shared/AzureCoreSharedEventSource.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.Tracing;
+
+namespace Azure.Core.Diagnostics
+{
+    [EventSource(Name = EventSourceName)]
+    internal sealed class AzureCoreSharedEventSource : EventSource
+    {
+        private const string EventSourceName = "Azure-Core-Shared";
```

Comment:
Curious why we actually need / care about a separate source for shared sources, though? Even though the sources are shared, do we feel it's not still part of "Core" as a concept? Or is this more of a case of not muddying the Core event waters?

/cc @pakrym

Reply:
Initially I had this in experimental, so this may not be necessary anymore.